README.md

```diff
@@ -163,10 +163,29 @@ ANALYTICS=YOUR_GOOGLE_TRACKING_ID
 
 
 ## Tokens tutorials
-### Discord Credentials
+### Discord Bot's Token
 <details>
- <summary>Click here to see Discord API Credentials tutorial!</summary>
- Soon!
+ <summary>Click here to see how to get your bot's token!</summary>
+
+1. Go to <a href="https://discordapp.com/developers/applications)">Discord Developer Portal</a>
+2. Your next step is to go over the menu on the left side of the screen and click “Bot”. It’s the icon that looks like a little puzzle piece.
```

Comment:
2.  At the top right of the screen, click "New application" and assign it a name. Next in the left part of the screen on the bar, find "Bot" then click it and find button "Add Bot"